sdks/python/apache_beam/io/gcp/spanner_wrapper.py

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the 'License'); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import uuid
+import apache_beam as beam
+
+try:
+  from google.cloud import spanner
+
+except ImportError:
+  spanner = None
+
+_LOGGER = logging.getLogger(__name__)
+
+class SpannerWrapper(object):
+  TEST_DATABASE = None
+  TEMP_DATABASE_PREFIX = 'temp-'
+  _SPANNER_CLIENT = None
+  _SPANNER_INSTANCE = None
+
+  def __init__(self, project_id = "apache-beam-testing", temp_database_id = None):
+    self._SPANNER_CLIENT = spanner.Client(project = project_id)
+    self._SPANNER_INSTANCE = self._SPANNER_CLIENT.instance("beam-test")
+    self.TEST_DATABASE = None
```

Comment:
Refactoring their usages in the rest of the class accordingly 
```suggestion
    self._spanner_client = spanner.Client(project = project_id)
    self._spanner_instance = self._spanner_client.instance("beam-test")
    self._test_database = None
```